packages/flutter/lib/src/material/navigation_bar.dart

```diff
@@ -217,16 +217,9 @@ enum NavigationDestinationLabelBehavior {
   onlyShowSelected,
 }
 
-/// Destination Widget for displaying Icons + labels in the Material 3
-/// Navigation Bars through [NavigationBar.destinations].
-///
-/// The destination this widget creates will look something like this:
-/// =======
-/// |
-/// |  ☆  <-- [icon] (or [selectedIcon])
-/// | text <-- [label]
-/// |
-/// =======
+/// A Material 3 [NavigationBar] destination.
+/// 
+/// Displays a label below an icon. Use with [NavigationBar.destinations].
```

Comment:
You could preserve the original diagram by enclosing it in triple backquotes.  Or you could generate a labeled diagram like the one that appears in the AppBar class API doc, using https://flutter.github.io/assets-for-api-docs/

Reply:
I don't think it's all that useful, given `NavigationBar` has a runnable code sample. I can add it to a new `See also` section